objectivec/GPBCodedInputStream.h

```diff
@@ -35,6 +35,34 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+CF_EXTERN_C_BEGIN
+
+/// GPBCodedInputStream exception name.
+extern NSString *const GPBCodedInputStreamException;
+
+/// NSError domain used for GPBCodedInputStream errors.
+extern NSString *const GPBCodedInputStreamErrorDomain;
+
+/// Error code for NSError with GPBCodedInputStreamErrorDomain.
+typedef NS_ENUM(NSInteger, GPBCodedInputStreamErrorCode) {
```

Comment:
Should I add a specific error domain (with it's own enum) for these errors? Or is it OK to mix them with the current ones?